modules/oe_newsroom_newsletter/tests/src/Functional/SubscribeBlockTest.php

```diff
@@ -61,8 +63,8 @@ public function testConfigurationForm(): void {
     $expected_languages = [
       'en' => 'English',
     ];
-    $this->assertEquals($expected_languages, $this->getOptions('Newsletter languages'));
-    $this->assertEquals($expected_languages, $this->getOptions('Default newsletter language'));
+    $this->assertEquals($expected_languages, $this->getSelectOptions('Newsletter languages'));
+    $this->assertEquals($expected_languages, $this->getSelectOptions('Default newsletter language'));
```

Comment:
I had to rename the method, because otherwise, phpstan still thinks it is the deprecated core method.